client/apps/shipping-label/view-wrapper-label.js

```diff
@@ -134,6 +145,10 @@ export class ShippingLabelViewWrapper extends Component {
 					>
 						{ translate( 'Track Package', 'Track Packages', { count: activeLabels.length } ) }
 					</Button>
+					<Suspense fallback={<div />}>
+						<LabelPurchaseModal orderId={ orderId } siteId={ siteId } />
+						<TrackingModal orderId={ orderId } siteId={ siteId } />
+					</Suspense>
 				</span>
```

Comment:
Same here with the `span`s. Block elements should be used instead.

Reply:
divs are better